website/project/views/register.py

```diff
@@ -151,10 +424,32 @@ def node_register_template_page_post(auth, node, **kwargs):
     schema = MetaSchema.find(
         Q('name', 'eq', template)
     ).sort('-schema_version')[0]
+
+    # Create the registration
     register = node.register_node(
         schema, auth, template, json.dumps(clean_data),
     )
 
+    if data['registrationChoice'] == 'Make registration public immediately':
```

Comment:
It would be better to use a non-human-readable representation for this. I.e.
```if data['registrationChoice'] == 'immediate':```
and
 ```elif data['registrationChoice'] == 'embargo':```

I'm guessing there is a html form somewhere dealing with this-- should the text change it would be better to only have to change the code in one place.